api-client/src/main/java/com/xing/api/data/messages/Conversation.java

```diff
@@ -26,135 +29,43 @@
 /**
  * Java representation of a Conversation.
  */
-public class Conversation implements Serializable {
+@AutoValue
+public abstract class Conversation implements Serializable {
```

Comment:
I can add it, no problem although I don´t see the need since those models are only intended to be created through moshi, right?